README.md

```diff
@@ -463,6 +464,18 @@ subscriber.on('update rate', (update) => {
 });
 ```
 
+Let's not forget to change the use of requester and responder in our conversion service and in our client to use segmentation key.
+
+conversion-service.js
+```js
+const responder = new cote.Responder({ name: 'currency conversion responder', key:'currecy' });
```

Comment:
There's a typo in the key here, instead of `currency`, it's `currecy` — however, I think calling the key `conversion` instead of `currency` here would be more explanatory.